source/ios/tools/IOSFeed/IOSFeed.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>IOSFeedNS</RootNamespace>
```

Comment:
>IOSFeedNS [](start = 19, length = 9)

nit: AdaptiveCards.Tools.IOSFeed

Reply:
I'm still seeing this in the latest iteration...

---
In reply to: [309844340](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r309844340) [](ancestors = 309844340)

Reply:
it changed from IOSFeed -> AdaptiveCards.Toos.IOSFeed :-)

---
In reply to: [310796198](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r310796198) [](ancestors = 310796198,309844340)